Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_net.cpp

```diff
@@ -6,6 +6,9 @@
 #include <cstdio>
 #include <cstdlib>
 #include <string>
+#include <arpa/inet.h>
```

Comment:
The change applies to all POSIX systems, not OpenBSD, so it would be better to use a not-Windows `#if` instead.